app/assets/javascripts/3dbio_viewer/src/webapp/view-models/Selection.ts

```diff
@@ -31,9 +31,16 @@ export interface FreeSelection extends BaseSelection {
     overlay: Array<DbItem>;
 }
 
+export type AllowedExtension = "pdb" | "cif" | "ent";
+
+export function getAllowedFileExtension(fileName: string) {
```

Comment:
[don't change it!] An eternal debate: `filename` vs `fileName`. Both are ok. We tend to use in the our code, though.